kasa/tests/fakeprotocol_smart.py

```diff
@@ -200,6 +201,49 @@ def _handle_control_child(self, params: dict):
             "Method %s not implemented for children" % child_method
         )
 
+    def _get_on_off_gradually_info(self, info, params):
```

Comment:
Maybe we should start moving fake module implementations into their own files to keep the fakeprotocol cleaner?

Reply:
Yes probably, it's getting a bit bloated.

Reply:
Yeah, having them separately would also make it easier to use them to create a fixture-based device simulator 😎

Reply:
Resolving for a future PR to clean up the fake protocol